spec/lib/simp/cli/commands/config_run_spec.rb

```diff
@@ -419,25 +456,26 @@
       summary_lines.delete_if { |line| line.chomp.empty? } # get rid of empty lines
 
       expected_lines = [
-        %r{Setting of \$simp_scenario in the simp environment's site.pp skipped}m,
+        %r{Setting of Puppet digest algorithm to sha256 for FIPS skipped}m,
```

Comment:
wasn't there talk about us bumping up to sha389?

Reply:
there is a ticket for that.  Not in the scope of the current ticket.